packages/volto/news/5607.feature

```diff
@@ -0,0 +1 @@
+Add ImageWidget with upload/drop/external and inline/widget capabilities @sneridagh @dobri1408
```

Comment:
The original was a little unclear, so this is my attempt to make it a little more clear.

```suggestion
Added `ImageWidget` with upload, drop, external, and inline capabilities. @sneridagh @dobri1408
```